src/DataViews/SamplingReportDataView.cpp

```diff
@@ -340,6 +339,34 @@ void SamplingReportDataView::SetStackEventsCount(uint32_t stack_events_count) {
   stack_events_count_ = stack_events_count;
 }
 
+std::string SamplingReportDataView::GetToolTip(int row, int column) {
+  if (column != kColumnInclusive && column != kColumnExclusive) {
+    return "";
+  }
+  const SampledFunction& function = GetSampledFunction(row);
+  uint32_t raw_count = {};
+  float percentage = {};
+  if (column == kColumnInclusive) {
+    raw_count = function.inclusive;
+    percentage = function.inclusive_percent;
+  } else {
+    raw_count = function.exclusive;
+    percentage = function.exclusive_percent;
+  }
+
+  orbit_statistics::BinomialConfidenceInterval interval =
```

Comment:
Hmm, I am thinking if it would make sense to also store the interval at the SampledFunction, to not always recalculate it. WDYT?

Reply:
I have considered such an option. My argument against is that:

1. The estimation is fast. 
2. The user will not hover too many cells in a rapid succession (and I don't think too many functions will be hovered in total), so this shouldn't cause a significant overhead. 
3. ... while storing a confidence interval for each `SampledFunction` may lead to a memory overhead.

Reply:
Fine for me. Thanks!